source/includes/_orders.md.erb

```diff
@@ -294,6 +294,54 @@ curl "<%= config[:host] %>/orders" \
 Получить все заказы пользователя можно, выполнив запрос:
 `GET <%= config[:host] %>/orders`
 
+## Применение промокода
+
+Чтобы применить промокод к заказу, необходимо выполнить запрос:
+
+`POST http://api.im.dev/v2/orders/#{NUMBER}/promotions`
+
+### Параметры запроса
+
+Параметр | Обязательный | Описание
+--------- | ------- | -----------
+promotion_code | Да | Промокод(число)
```

Comment:
> Промокод(число)

Then `C#` developers will hardcode `Integer/Float` datatype and our users won't be able to enter promocode 😄